### PR TITLE
[WIP] BT led: color more keys

### PR DESCRIPTION
**Pull request by hdhoang**
Besides stock keys, we use QWER to save host and BSpace for BtOn.

I have some more planned:
- [ ] red ASDF for delete host (since we haven't implemented stock's A anyway)
- [ ] Something for RCtrl
- [x] red 0 for unknown BLE status (currently blue)

I would like your comment on these.

### CODE REVIEW

**Comment by ah-** [comment]
Cool, this looks nice. Have you tried if it works on real hardware?

File: src/led.rs [hunk 1]
```diff
@@ -109,21 +109,29 @@ where
...
+            // Select host
             KeyIndex::N1 as u8,     0xff, 0x00, 0x00, LedMode::Flash as u8,
             KeyIndex::N2 as u8,     0xff, 0x00, 0x00, LedMode::On as u8,
             KeyIndex::N3 as u8,     0xff, 0x00, 0x00, LedMode::On as u8,
             KeyIndex::N4 as u8,     0xff, 0x00, 0x00, LedMode::On as u8,
+            // Save host
+            KeyIndex::Q as u8,      0x00, 0x00, 0xff, LedMode::On as u8,
+            KeyIndex::W as u8,      0x00, 0x00, 0xff, LedMode::On as u8,
+            KeyIndex::E as u8,      0x00, 0x00, 0xff, LedMode::On as u8,
+            KeyIndex::R as u8,      0x00, 0x00, 0xff, LedMode::On as u8,
```
Comment (ah-) [inline src/led.rs]: I never quite understood how those number keys are meant to work on the stock fw. If you press a number, what does it do? Connect to that host or save?

Comment (hdhoang) [reply src/led.rs]: The obins docs says `Fn+1` saves the current host to number 1, after that `1` reconnects to that saved host. Can you try capturing the MCU<->BLE traffic for those actions?

Admittedly I haven't groked the switching functionality yet. I'm not even sure which host is saved in which slot now. Most of the time I'm switching between 2 computers side-by-side, then at other time I use a dual-boot laptop, and another desktop with the obins dongle. Then there's my oreo phone which doesn't remember the anne for long (!?).

With your fw, I often bring up the hosts' bluetooth control panels to check where it's connected. Using the `Connect` keys can either (a) disconnect current, then reconnect to current stably (b) rapidly disconnect and reconnect from the current host (happens with key 1/slot 0 the most in my daily usage) (c) disconnect, then connect to the other computer (my phone doesn't remember the anne anyway).

All that said, this PR is coloring the keys with functions in BT layer. The actual functionality dance should go into proper documentation.

**Comment by hdhoang** [comment]
Yep, after fixing the length field in `payload`, I have the keys light up as specified! (I use dvorak so the bluetooth key is on `KeyIndex::N` though, please look out if I committed that by mistake)

**Pull request merged**